### ISSUE TITLE
Execution driver missing from `docker info` output

### ISSUE THREAD

**Issue by shin-** [docker/docker#24461]
**Output of `docker version`:**

```
$ docker version
Client:
 Version:      1.12.0-rc3
 API version:  1.24
 Go version:   go1.6.2
 Git commit:   91e29e8
 Built:        Sat Jul  2 00:38:44 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.0-rc3
 API version:  1.24
 Go version:   go1.6.2
 Git commit:   91e29e8
 Built:        Sat Jul  2 00:38:44 2016
 OS/Arch:      linux/amd64

```

**Output of `docker info`:**

```
$ docker info
Containers: 107
 Running: 0
 Paused: 0
 Stopped: 107
Images: 386
Server Version: 1.12.0-rc3
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 682
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: null host overlay bridge
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: apparmor seccomp
Kernel Version: 4.4.0-28-generic
Operating System: Ubuntu 16.04 LTS
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 7.499 GiB
Name: yuna
ID: YZQG:AKYJ:JQXJ:CEHT:62DP:PMWA:S43W:5X47:SSOE:UXGG:XHWS:555F
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Username: joffrey
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Insecure Registries:
 127.0.0.0/8

```

**Additional environment details (AWS, VirtualBox, physical, etc.):** Ubuntu 16.04 desktop

**Steps to reproduce the issue:**
1. Run `docker info` command

**Describe the results you received:**
"Execution driver" is missing from the output

**Describe the results you expected:**
"Execution driver: native-0.2" is present in the output 

**Additional information you deem important (e.g. issue happens only occasionally):**
In the JSON payload of the `/info` endpoint, the `ExecutionDriver` key is present with an empty string `""` value.
```javascript
{
  "ContainersPaused": 0, 
  "Labels": null, 
  "DefaultRuntime": "runc", 
  "CgroupDriver": "cgroupfs", 
  "ContainersRunning": 0, 
  "NGoroutines": 30, 
  "Swarm": {
    "CACertHash": "", 
    "Managers": 0, 
    "ControlAvailable": false, 
    "NodeID": "", 
    "Error": "", 
    "RemoteManagers": null, 
    "LocalNodeState": "inactive", 
    "Nodes": 0
  }, 
  "LoggingDriver": "json-file", 
  "OSType": "linux", 
  "HttpProxy": "", 
  "Runtimes": {
    "runc": {
      "path": "docker-runc"
    }
  }, 
  "DriverStatus": [
    [
      "Root Dir", 
      "/var/lib/docker/aufs"
    ], 
    [
      "Backing Filesystem", 
      "extfs"
    ], 
    [
      "Dirs", 
      "682"
    ], 
    [
      "Dirperm1 Supported", 
      "true"
    ]
  ], 
  "OperatingSystem": "Ubuntu 16.04 LTS", 
  "Containers": 107, 
  "HttpsProxy": "", 
  "BridgeNfIp6tables": true, 
  "MemTotal": 8052154368, 
  "Driver": "aufs", 
  "IndexServerAddress": "https://index.docker.io/v1/", 
  "ClusterStore": "", 
  "ExecutionDriver": "", 
  "SystemStatus": null, 
  "OomKillDisable": true, 
  "ClusterAdvertise": "", 
  "SystemTime": "2016-07-08T13:07:15.079350072-07:00", 
  "Name": "yuna", 
  "CPUSet": true, 
  "RegistryConfig": {
    "InsecureRegistryCIDRs": [
      "127.0.0.0/8"
    ], 
    "IndexConfigs": {
      "docker.io": {
        "Official": true, 
        "Name": "docker.io", 
        "Secure": true, 
        "Mirrors": null
      }
    }, 
    "Mirrors": null
  }, 
  "SecurityOptions": [
    "apparmor", 
    "seccomp"
  ], 
  "ContainersStopped": 107, 
  "NCPU": 4, 
  "NFd": 23, 
  "Architecture": "x86_64", 
  "KernelMemory": true, 
  "CpuCfsQuota": true, 
  "Debug": false, 
  "ID": "YZQG:AKYJ:JQXJ:CEHT:62DP:PMWA:S43W:5X47:SSOE:UXGG:XHWS:555F", 
  "IPv4Forwarding": true, 
  "KernelVersion": "4.4.0-28-generic", 
  "BridgeNfIptables": true, 
  "NoProxy": "", 
  "ServerVersion": "1.12.0-rc3", 
  "CpuCfsPeriod": true, 
  "ExperimentalBuild": false, 
  "MemoryLimit": true, 
  "SwapLimit": false, 
  "Plugins": {
    "Volume": [
      "local"
    ], 
    "Network": [
      "bridge", 
      "null", 
      "host", 
      "overlay"
    ], 
    "Authorization": null
  }, 
  "Images": 386, 
  "DockerRootDir": "/var/lib/docker", 
  "NEventsListener": 0, 
  "CPUShares": true
}

```

**Comment by hqhq** [comment]
There is no execution driver now, I cleaned this up in https://github.com/docker/docker/pull/24501

**Comment by coolljt0725** [comment]
Fixed in #24501
